src/components/utils/test/scope_guard_test.cc

```diff
@@ -31,6 +31,7 @@
  */
 
 #include "gmock/gmock.h"
+#include "gtest/gtest.h"
```

Comment:
@OHerasym Why do you need both gmock.h and gtest.h? gmock.h already contains gtest.h